perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -481,8 +463,26 @@ sub bin_services_for_address {
 
     @site_services_filtered = $self->service_sort(@site_services_filtered);
 
+    # Garden subscription.
+    # This call removes Whitespace service if there is no contract in Agile.
     $property->{garden_current_subscription}
         = $self->garden_current_subscription(\@site_services_filtered);
+    $property->{has_garden_subscription} = 1
+        if $property->{garden_current_subscription};
+
+    # To begin with we assume the property is eligible to sign up to GGW...
+    $property->{garden_signup_eligible} = 1;
+    # unless it's got a parent property,
+    # or no services of its own
+    # or it already has a subscription in Agile
+    # or it has sacks
+    if (   $property->{parent_property}
+        || !@site_services_filtered
+        || $property->{garden_current_subscription}
```

Comment:
I feel like `$property->{has_garden_subscription}` would be more semantically correct here, though it obviously doesn't make any material difference!